content/updates/2020-10/6-Job-and-process-tracker-improvements.md

```diff
@@ -1,22 +1,40 @@
 ---
-pim_announcement_alt_img: ""
+pim_announcement_alt_img: "TODO"
 pim_announcement_audience:
 - EE
 - CE
 ---
 
 # Jobs and process tracker improvements
 
-TODO
+Follow the estimated time left for your ongoing jobs, quickly jump to the process tracker once you launched a quick export, and enjoy some enhancements directly inside your process tracker grid.
 
 ## Progress bar and time left for your ongoing jobs
-::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
+::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/TODO.html"
 :::
 
+In the process tracker, in the detail of each ongoing job, we now compute an estimation of the time left for each job step. A progress bar is also displayed.
```

Comment:
```suggestion
From now on, you will be able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. 
```
Do you know if this applies to all jobs? imports and exports?

Reply:
Import and export. @felix-couprie Can you confirm?

Reply:
Hello Nolwenn and Laure!
Yes, it applies to import and export jobs that are listed in the Notion page. Note that we way not be able to have the progress for every step when tracking a custom job.

Reply:
No future in sentences, @LaureBro. I suggest:
```suggestion
You are now able to see an estimation of the remaining time and a progress bar for all your ongoing jobs. 
```